lib/Converter.php

```diff
@@ -116,7 +116,7 @@ private static function convertBlockFromRaw(object $rawBlock): ContentBlock
             $rawBlock->text,
             $characterList,
             $rawBlock->depth,
-            json_decode(json_encode($rawBlock->data), true) ?: [],
+            isset($rawBlock->data) && json_decode(json_encode($rawBlock->data), true) ?: [],
```

Comment:
This can be shortened with the use of the [null coalescing operator](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op).

```suggestion
            json_decode(json_encode($rawBlock->data ?? []), true),
```